### PR TITLE
Record variables to job result

### PR DESCRIPTION
**Pull request by ErikZigo**
Fixes: https://keboola.atlassian.net/browse/SOX-391

Tohle je hlavní PR, závislý na:
- configuration resolver  https://github.com/keboola/platform-libraries/pull/160
- internal api client https://github.com/keboola/job-queue/pull/348
- docker bundle https://github.com/keboola/docker-bundle/pull/721

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -22,10 +22,10 @@
         "ext-pcntl": "*",
         "aws/aws-sdk-php": "3.258",
         "keboola/api-error-control": "^3.10",
-        "keboola/configuration-variables-resolver": "^4.0",
-        "keboola/dockerbundle": "dev-odin-library-3",
+        "keboola/configuration-variables-resolver": "dev-erik-SOX-391 as 4.1.1",
+        "keboola/dockerbundle": "dev-erik-SOX-391",
         "keboola/input-mapping": "^18.3",
-        "keboola/job-queue-internal-api-php-client": "^18.2",
+        "keboola/job-queue-internal-api-php-client": "dev-erik-SOX-391 as 18.4.0",
```
Comment (MiroCillik) [inline composer.json:28]: Nezapomenout

**Review by MiroCillik — APPROVED** [review]
Nezapomenout updatnout zavislosti, jinak LGTM

File: composer.json (hunk 1 above)
Comment (ErikZigo) [reply composer.json]: @MiroCillik verze jsem upravil na normalni releasnuty + odin bundle

**Review by MiroCillik — APPROVED** [review]
👍

**Pull request merged**